src/app/auth/components/auth/auth.component.scss

```diff
@@ -58,7 +58,7 @@
 .image {
   position: relative;
   height: 100%;
-  background: url('../../../../assets/img/sign_in_image.png');
```

Comment:
Can you give this a bit of a clearer name like `legacy_lab_sign_in_image.png`? Just so it's easier to track and understand its purpose without opening the file.